themes/10up-theme/includes/blocks/example-block/edit.js

```diff
@@ -4,16 +4,19 @@
 import { __ } from '@wordpress/i18n';
 import { RichText, useBlockProps } from '@wordpress/block-editor';
 
+// Importing the block's css via JS will enable hot reloading for css
+import './index.css';
```

Comment:
As part of this, we should rename the `index.css` file to `editor.css`. It currently is confusing and especially now with this new comment it makes it sound as if all the block styles should be written in that file.

Reply:
Sure, I'll rename this to editor.css

Reply:
Done!